assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -117,7 +123,7 @@ const CartLineItemRow = ( { lineItem } ) => {
 				<div className="wc-block-cart-item__product-name">{ name }</div>
 				{ lowStockBadge }
 				<div className="wc-block-cart-item__product-metadata">
-					{ description }
+					<div dangerouslySetInnerHTML={ { __html: description } } />
```

Comment:
Remember we introduced a truncated description to the cart API called `summary`. That is what should be used here.

Reply:
Aha! Now using `summary` 5014402a702edbc4baeae7e429cac2850757ce05